cli/integration_tests/single_package/dry-run.t

```diff
@@ -8,7 +8,7 @@ Check
   Tasks to Run
   build
     Task                   = build                                                                                                       
-    Hash                   = 7bf32e1dedb04a5d                                                                                            
+    Hash                   = 46d68c621aadbb40                                                                                            
```

Comment:
Can you describe what it is that is causing these hashes to change? Do we need to bump the global cache key?

Reply:
It's because the Workspace Directories are part of the task hash, and in this case, the Root Workspace changed it's representation of a Directory (from `""` to `"."`)

https://github.com/vercel/turbo/blob/bbdb5e592cae2a557a4c6adfb423216d461bc09d/cli/internal/taskhash/taskhash.go#L297

<details>
<summary>
Only the Single Package hashes were changed here, since their workspace is the root directory. Root Tasks would also bust their cache,but I'm guessing we don't have any dry run tests for those yet.
</summary>

```
$ git d main --stat integration_tests/                                       
cli/integration_tests/single_package/dry-run.t           |  4 ++--
cli/integration_tests/single_package/run.t               |  4 ++--
cli/integration_tests/single_package_deps/dry-run.t      |  8 ++++----
cli/integration_tests/single_package_deps/run.t          | 12 ++++++------
cli/integration_tests/single_package_no_config/dry-run.t |  4 ++--
cli/integration_tests/single_package_no_config/run.t     |  4 ++--
6 files changed, 18 insertions(+), 18 deletions(-)
```
</details>

I was a little worried about this too and meant to ask if we have a policy around cache busting in minor releases (another reason why I feel like a release branch would be a good idea, btw).

> Do we need to bump the global cache key?

What did you mean by this? I know [there's a key](https://github.com/vercel/turbo/blob/bbdb5e592cae2a557a4c6adfb423216d461bc09d/cli/internal/run/global_hash.go#L20), but what would bumping it do in this situation?

Reply:
Reverted the thing that made the taskhash change